src/tests/ftest/dfuse/mu_perms.py

```diff
@@ -369,7 +369,7 @@ def _verify(use_il, expected_il_messages, expect_der_no_perm):
             self.log.info('Verify - no perms - not using IL')
             _verify(use_il=False, expected_il_messages=0, expect_der_no_perm=False)
             self.log.info('Verify - no perms - using IL')
-            _verify(use_il=True, expected_il_messages=1, expect_der_no_perm=False)
+            _verify(use_il=True, expected_il_messages=2, expect_der_no_perm=False)
```

Comment:
This change was cherry-picked from https://github.com/daos-stack/daos/commit/b33a13218078f7bcdcb0032e9927b81cd55ba3be
It should merge cleanly now